src/app/components/person/person.component.html

```diff
@@ -1,5 +1,11 @@
 <div class="myContainer">
   <app-spinner name="person"></app-spinner>
+  <!-- for fake route -->
+  <div *ngIf="personPosition">
+    <app-persons-header [style.display]="perHeaderVis" [PersonType]="personPosition"></app-persons-header>
```

Comment:
PersonType with uppercase? Is that class?
Improve [style.display]="perHeaderVis"